blackbird_python/blackbird/program.py

```diff
@@ -314,25 +313,47 @@ def serialize(self):
 
         script = ["name {}".format(self.name), "version {}".format(self.version)]
 
-        if self.target["name"] is not None:
-            array_insert += 1
-            options = ""
+        # add target and type to the script
+        for n, md in [("target", self.target), ("type", self.programtype)]:
```

Comment:
What does `md` refer to here? Perhaps this variable could have a more descriptive name

Reply:
Oh, I don't remember. Good point, I'll update it!